README.md

```diff
@@ -30,6 +30,8 @@ yarn install
 yarn start
 ```
 
+You can read more in the [UI README](./src/ui/README.md).
```

Comment:
Didn't you delete the Readme file in #8 ?

Reply:
Yes, this is a rebase issue, will remove this line again

Reply:
Fixed, BTW I added contributors field on the package.json, add yourself! @BlackMcDevil